webhook/resources/volume/validator.go

```diff
@@ -491,6 +504,11 @@ func validateReplicaCount(dataLocality longhorn.DataLocality, replicaCount int)
 			return werror.NewInvalidError(fmt.Sprintf("number of replica count should be 1 when data locality is %v", longhorn.DataLocalityStrictLocal), "")
 		}
 	}
+	if cloneMode == longhorn.CloneModeLinkedClone {
+		if replicaCount != 1 {
+			return werror.NewInvalidError(fmt.Sprintf("number of replica count must be 1 when clone mode %v", longhorn.CloneModeLinkedClone), "")
+		}
+	}
```

Comment:
Is it possible to remove the limitation in the future?

Reply:
I think we should not remove it. This is by design that linked-clone volume is a short-live, no replication, for backup solution to quickly read data and then destroy it.